### PR TITLE
Link to the new documentation in the readme

### PR DESCRIPTION
**Pull request by rchl**
Also remove links to specific languages as that list can easily get out
of sync with documentation.

### CODE REVIEW

**Comment by rchl** [comment]
I'm not sure about the removal of the languages list because that can have a negative impact on discovery / SEO.

If we want to keep it then probably should keep it in a slightly different form (just mention it as examples, without links).

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
+:information_source: Note that the `st4000-exploration` branch corresponds to the ST4 version of LSP. If you are looking for information on ST3 version, switch to [master branch](/LSP/tree/master).
```
Comment (rchl) [inline README.md:1]: Not sure if that link to master branch will work.

Comment (predragnikolic) [reply README.md:1]: https://github.com/sublimelsp/LSP/blob/72aae217a2b4e8dc65e8877f9fe91016ea8b0ca7/README.md 

looks like that it doesn't work

File: README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
+:information_source: Note that the `st4000-exploration` branch corresponds to the ST4 version of LSP. If you are looking for information on ST3 version, switch to [master branch](https://github.com/sublimelsp/LSP/tree/master).
```
Comment (rwols) [inline README.md:1]:
```suggestion
:information_source: You are reading the README of the `st4000-exploration` branch. This branch targets ST4. If you are looking for the ST3 version, switch to the [master branch](https://github.com/sublimelsp/LSP/tree/master).
```

**Review by rwols — CHANGES_REQUESTED** [review]

**Review by rwols — APPROVED** [review]

**Pull request merged**